README.md

```diff
@@ -36,12 +36,12 @@ for
 and
 [azure-identity-binding](./helm/ffc-demo-claim-service/templates/azure-identity-binding.yaml).
 
-| Name                               | Description                                                                                  |
-| ---------------------------------- | -------------------------------------------------------------------------------------------- |
-| MESSAGE_QUEUE_HOST                 | Azure Service Bus hostname, e.g. `myservicebus.servicebus.windows.net`                       |
-| MESSAGE_QUEUE_PASSWORD             | Azure Service Bus SAS policy key                                                             |
-| MESSAGE_QUEUE_SUFFIX               | Developer specific queue suffix to prevent collisions, only required for local development   |
-| MESSAGE_QUEUE_USER                 | Azure Service Bus SAS policy name, e.g. `RootManageSharedAccessKey`                          |
+| Name                   | Description                                                                                |
+| ----                   | -----------                                                                                |
```

Comment:
Does this whitespace need to be here?

Reply:
I use a plugin to format tables and keeping the columns equal width is one of the things it does, improving readability. Removing the `-` from the heading makes the column fit to the content when it is formatted as opposed to having to be at least as wide the set of `-` that were initially created.